README.markdown

```diff
@@ -28,7 +28,11 @@ Generates or modifies settings for your application, based on prompts. This is t
 
     grove config
 
-You can see options for this command by running:
+You can pass the `environment` you want to override. By default it modifies the `local` environment. You could change this to `development`, for example:
+
+    grove config development
+
+You can see all options for this command by running:
```

Comment:
I recall @patrickmcelwee said that React allows .env, .env.local, .env.development, and .env.development.local, in order of increasing priority. If that is true, we need to support this, for instance by allowing end users to run: `grove config` (-> .env), `grove config --local` (-> .env.local), `grove config node_env` (-> .env.node_env), and `grove config --local node_env` (-> .env.node_env.local)

Reply:
@mariannemyers / @withjam, can either of you confirm this behavior in React, and comment?

Reply:
Mind though, Gradle does not allow same kind of granularity. There is no gradle-env-local.properties.. only -local or -env..